website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,202 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another WSL distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, Podman Desktop configures Podman in a dedicated Windows Subsystem for Linux (WSL) distribution: the Podman Machine.
+It also configures the Windows Podman client to communicate with the Podman Machine.
+But it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the Ubuntu distribution of Linux, to walk you through the steps to configure your WSL distribution:
+
+1. Install the Podman client that will initiate requests to the remote Podman.
+2. Configure access to the Podman Machine defined by Podman Desktop.
+3. Assign the remote destination.
+4. Test the connection.
+
+## Installing the Podman client
+
+On your WSL distribution, install the Podman client to send requests to the remote Podman Machine.
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. Install the Podman client:
+
+   ```shell-session
+   $ sudo apt install podman
+   ```
+
+   The package might set up local Podman resources, but you are not actively using them.
+
+## Giving access to the Podman Machine defined by Podman Desktop
+
+
+On your WSL distribution, give access to the remote Podman machine to your user, by assigning group membership.
+
+It will allow your user to access a special file (a socket), that is the communication channel to the Podman Machine.
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. Add your user to the `10` group:
+
+   ```shell-session
+   $ sudo usermod --append --groups 10 your_user
+   ```
+
+1. Exit to apply the new group membership:
+
+   ```shell-session
+   $ exit
+   ```
+
+1. Start a new WSL session:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. List your user groups:
+
+   ```shell-session
+   $ groups
+   ```
+
+   On the default Ubuntu WSL, the `uucp` group is in the list.
+
+   <details>
+   <summary>
+   On a custom WSL distribution, the group name might be different: find the required group name:
+   </summary>
+   <div>
+   The required group id is the same on any WSL distribution.
+
+   However, the group name might be different on a custom WSL distribution.
+
+   On the default Ubuntu distribution:
+
+   - Rootful Podman: GID `10` name is `uucp`.
```

Comment:
You might want to mention that this group is called "wheel" on Fedora, otherwise it might not make any sense?

https://en.wikipedia.org/wiki/UUCP

Usually it is more explicit to add a new "podman" group, but Podman choose to reuse the old BSD admin group.

https://en.wikipedia.org/wiki/Wheel_(computing)

Reply:
The group is normally called "sudo" in ubuntu, which is why the "uucp" group needed to be added explicitly

```
## Allow root to run any commands anywhere 
root	ALL=(ALL) 	ALL

## Allows members of the 'sys' group to run networking, software, 
## service management apps and more.
# %sys ALL = NETWORKING, SOFTWARE, SERVICES, STORAGE, DELEGATING, PROCESSES, LOCATE, DRIVERS

## Allows people in group wheel to run all commands
%wheel	ALL=(ALL)	ALL
```

```
# User privilege specification
root	ALL=(ALL:ALL) ALL

# Members of the admin group may gain root privileges
%admin ALL=(ALL) ALL

# Allow members of group sudo to execute any command
%sudo	ALL=(ALL:ALL) ALL
```

Reply:
Looks good, need Windows to be able to test the other things so will leave that for other reviewers